### PR TITLE
Throw error if openBrowser is called multiple times #818

### PR DESCRIPTION
**Pull request by suganya-sangith**
@nehashri I have used the chromeProcess to check whether the browser is already open, please review the code and let me know if there are any changes

### CODE REVIEW

**Comment by suganya-sangith** [comment]
@nehashri Raised a PR for this please review and let me know if there are any changes

File: lib/taiko.js [hunk 1]
```diff
@@ -198,6 +198,11 @@ module.exports.openBrowser = async (options = { headless: true }) => {
       'Invalid option parameter. Refer https://taiko.gauge.org/#parameters for the correct format.',
     );
   }
+
+  if (chromeProcess && !chromeProcess.killed) {
+    throw new Error('openBrowser cannot be called again as there is a chromium instance open (closeBrowser is not yet called)');
```
Comment (Apoorva-GA) [inline lib/taiko.js]: Can you change the message to `OpenBrowser cannot be called again as there is a chromium instance open.';

**Comment by Apoorva-GA** [comment]
@suganya-sangith Also there are some conflicts. Can you please resolve them?

File: lib/taiko.js (hunk 1 above)
Comment (suganya-sangith) [reply lib/taiko.js]: @Apoorva-GA Sure will make the changes

**Comment by suganya-sangith** [comment]
@Apoorva-GA I have made the changes you mentioned, could you please rerun the checks?

**Comment by Apoorva-GA** [comment]
@suganya-sangith Thanks for the PR!

**Pull request merged**